website/docs/contribution/contribute_to_warp/contribution-guidelines.md

```diff
@@ -0,0 +1,93 @@
+
+#### 2. Read guidelines for versioning, changelog syntax, and WARP Documentation style.
+
+Our testing infrastructure relies on specific [versioning](https://broadinstitute.github.io/warp/docs/About_WARP/VersionAndReleasePipelines) and [changelog syntax](https://broadinstitute.github.io/warp/docs/contribution/contribute_to_warp/changelog_style). Pipeline changes must be documented in the changelog with the appropriate syntax.
+
+Similarly, if you plan to contribute to WARP Documentation, read the [WARP Documentation style guide](https://broadinstitute.github.io/warp/docs/contribution/contribute_to_warp_docs/doc_style) first, as it contains helpful formatting tips.
+
+#### 3. Make a branch or fork off the WARP develop branch.
+
+If you have WARP repository permissions, GitHub will allow you to make a branch off the WARP **develop** branch when you’re ready to contribute. For more information on how to make a branch, read the GitHub Docs [instructions for branching](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/managing-branches#creating-a-branch). 
+
+If you **do not** have WARP permissions (i.e. you can’t make a branch off of the develop branch), make a fork of the WARP repository following the GitHub Docs [instructions for forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+![](fork.png)
+
+#### 4. Make your updates on your WARP branch or fork. 
+As your work progresses, make commits to your WARP branch or fork. 
+
+#### 5. Assess and make updates to accompanying pipeline files.
+
+Updates to pipeline code, Docker images, or reference files might require coordinating additional updates both in WARP, such as modifying accompanying changelogs and pipeline overviews, or outside of WARP, such as uploading Docker images or reference files to public cloud repositories.  
+
+* All changes that affect pipeline workflows must be recorded in the pipeline(s) changelog(s) using semantic versioning. An explanation of this can be found in the [changelog style guide](https://broadinstitute.github.io/warp/docs/contribution/contribute_to_warp/changelog_style). 
+
+* When you update the changelog, the pipeline’s WDL workflow version number must also match the changelog entry to ensure the updates pass the WARP testing process (this is shown in the [Example Contribution](#example-contribution-process)).
+
+* If you make any changes to files hosted in cloud repositories, like pipeline Docker images or reference files, coordinate with the WARP team during or before the review process to push the updated files to cloud storage. 
+
+* Flag or make necessary updates to the accompanying pipeline documentation such as the pipeline [overviews in WARP Documentation](https://broadinstitute.github.io/warp/). 
```

Comment:
A little unclear here.  You mean to have them not to forget to add changes to the overview?

Reply:
I updated it to be more explicit to change the documentation if necessary.